applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_server.c

```diff
@@ -659,49 +663,43 @@ int unicast_server_enable(le_audio_receive_cb recv_cb)
 	for (int i = 0; i < ARRAY_SIZE(caps); i++) {
 		ret = bt_pacs_cap_register(caps_dirs[i], &caps[i]);
 		if (ret) {
-			LOG_ERR("Capability register failed");
+			LOG_ERR("Capability register failed: %d", ret);
 			return ret;
 		}
 	}
 
-	if (channel == AUDIO_CH_L) {
-		csip_param.rank = CSIP_HL_RANK;
+	if (IS_ENABLED(CONFIG_BT_AUDIO_RX)) {
+		ret = bt_pacs_set_supported_contexts(BT_AUDIO_DIR_SINK, AVAILABLE_SINK_CONTEXT);
 
-		ret = bt_pacs_set_location(BT_AUDIO_DIR_SINK, BT_AUDIO_LOCATION_FRONT_LEFT);
 		if (ret) {
-			LOG_ERR("Location set failed");
+			LOG_ERR("Supported context set failed. Err: %d", ret);
```

Comment:
Nitpick: probably we can unify the format like line 666.

Reply:
I changed them all to be like this instead. Better to print the error code directly